app/models/core_data_connector/location.rb

```diff
@@ -1,9 +0,0 @@
-module CoreDataConnector
```

Comment:
This seems unrelated to the PR but I also don't recognize it - is this dead code that's being cleaned up?